### PR TITLE
Commit for Access Token

### PR DESCRIPTION
**Pull request by Treesa-James**
To check the Authorization of a token by sending a sample request.

### CODE REVIEW

File: tests/IntegrationTest.php [hunk 1]
```diff
@@ -18,11 +21,30 @@ final class IntegrationTest extends TestCase
...
+            'access_token' =>Config::HUBSPOT_ACCESS_TOKEN
         ];
     }
+    
+    function testUnauthorizedAccess(){
+      $client = Factory::createWithAccessToken($this->oauthConfig['access_token']); 
+      try {
+        $apiResponse = $client->crm()->companies()->basicApi()->getPage(10, false); 
+        $responseData = json_decode($apiResponse, true);
+        $responseData = json_decode($apiResponse, true);
```
Comment (arun-experion) [inline tests/IntegrationTest.php:33]: Correct this code duplication

**Review by arun-experion — COMMENTED** [review]
Review comments added

File: tests/IntegrationTest.php (hunk 1 above)
Comment (Treesa-James) [reply tests/IntegrationTest.php]: Correct duplication of response data

**Review by arun-experion — APPROVED** [review]
Code review completed

**Pull request merged**